x/fswap/types/params.go

```diff
@@ -1,21 +1,56 @@
 package types
 
 import (
+	"errors"
+	fmt "fmt"
+	"strings"
+
 	"gopkg.in/yaml.v2"
+
+	sdk "github.com/Finschia/finschia-sdk/types"
+)
+
+const (
+	DefaultNewCoinDenom string = "PDT"
```

Comment:
How about using configuration injected within calling `NewKeeper()`? The application may change the literal without modifying the SDK. This could be also applied to the exchanging rate and the old denom.

Reply:
Since `DefaultNewCoinDenom` is bound with fswap module, and we will not change it after launch, I have placed it in the params.
About the exchange rate and oldcoindenom, I will inject them as config.

Reply:
Actually, it would be opposite. The argument handed over by `NewKeeper()` would live in the application, and the value itself would be a literal (i.e. constant). The parameter, on the other hand, could be changed by a proposal.
If your "params" means a constant in the SDK, I would also prefer the `NewKeeper()` method, because the application may use a different value in their business logic.